src/test/java/dev/jbang/source/TestSource.java

```diff
@@ -322,8 +322,8 @@ void testExtractGroovyOptions() {
 
 		assertEquals(Arrays.asList("--enable-preview"), s.getCompileOptions());
```

Comment:
why the order flip? (no, probably fine - just curious as feel weird)

also ...why does it have to --enable-preview....?

Reply:
There's actually not a real reason. When refactoring the code I felt that the COMPILE/RUNTIME_OPTIONS were the "real" options and the JAVA[C]_OPTIONS were the legacy, old, options. So the "real" options should go first. But that was me assuming there was really no difference, especially because who would use both (real and old) options in the same file?
But if you think there could be an issue I'll change it back.

Reply:
ok, so should be fine/ not have impact.